doc/CONFIGURATION.md

```diff
@@ -60,7 +60,8 @@ The available keys and their accepted values are reported in the table below.
 | log_connections | `off` | Bool | No | Log connects |
 | log_disconnections | `off` | Bool | No | Log disconnects |
 | blocking_timeout | 30 | Int | No | The number of seconds the process will be blocking for a connection (disable = 0) |
-| idle_timeout | 0 | Int | No | The number of seconds a connection is been kept alive (disable = 0) |
+| idle_timeout | 0 | Int | No | The number of seconds a connection is been kept alive since becomes idle (disable = 0) |
+| max_connection_age | 0 | Int | No | The number of seconds a connection is been kept alive since starts (disable = 0) |
```

Comment:
"... since it times out"

Reply:
If I get it right, 
- `idle_timeout` is "The number of seconds a connection can remain idle before it is automatically closed"
- `max_connection_age` is "The number of seconds a connection can remain active before it is automatically closed if it is not being used"

Reply:
It is the maximum seconds that a connection will live - it will be checked upon returned to the pool, or during idle timeout

Reply:
Thanks for the feedback! I updated the `max_connection_age` docs and kept the `idle_timeout` docs untouched.

Checking upon returned to the pool is added to `pgagroal_return_connection`.

Both ways of checking are tested using a basic setup.